app/javascript/components/close_contact/CloseContact.js

```diff
@@ -128,141 +151,139 @@ class CloseContact extends React.Component {
+              />
+              <datalist id="assigned_users">
+                {this.props.assigned_users?.map(num => {
+                  return (
+                    <option value={num} key={num}>
+                      {num}
+                    </option>
+                  );
+                })}
+              </datalist>
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['assigned_user']}
+              </Form.Control.Feedback>
+            </Form.Group>
+          </Row>
+          <Row>
+            <Form.Group as={Col}>
+              <Form.Label htmlFor="notes" className="nav-input-label">
+                Notes
+                {schema?.fields?.notes?._exclusive?.required && '*'}
+              </Form.Label>
+              <Form.Control
+                id="notes"
+                as="textarea"
+                rows="5"
+                className="form-square"
+                value={this.state.notes || ''}
+                placeholder={this.closeContactNotePlaceholder}
+                maxLength="2000"
+                onChange={this.handleChange}
+              />
+              <Form.Label className="notes-character-limit"> {2000 - this.state.notes.length} characters remaining </Form.Label>
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['notes']}
+              </Form.Control.Feedback>
+            </Form.Group>
+          </Row>
         </Modal.Body>
         <Modal.Footer>
           <Button variant="secondary btn-square" onClick={toggle}>
             Cancel
           </Button>
-          <Button
-            variant="primary btn-square"
-            onClick={submit}
-            disabled={
-              !(this.state.first_name || this.state.last_name || this.state.email || this.state.primary_telephone || this.state.notes) || this.state.loading
-            }>
-            {this.props.close_contact.id ? 'Update' : 'Create'}
+          <Button variant="primary btn-square" onClick={submit} disabled={this.state.disableCreate || this.state.loading}>
+            <span data-for="create-tooltip" data-tip="" className="ml-1">
+              {this.props.close_contact.id ? 'Update' : 'Create'}
+            </span>
           </Button>
+          {/* Typically we pair the ReactTooltip up directly next to the mount point. However, due to the disabled attribute on the button */}
+          {/* above, this Tooltip should be placed outside the parent component (to prevent unwanted parent opacity settings from being inherited) */}
+          {/* This does not impact component functionality at all. */}
+          {this.state.disableCreate && (
+            <ReactTooltip id="create-tooltip" multiline={true} place="top" type="dark" effect="solid" className="tooltip-container text-left">
+              Please enter one of the following: First Name, Last Name, Phone Number or Email
```

Comment:
Comma after "Phone Number"

Reply:
A fellow supporter of the oxford comma, I see

Reply:
Okay, will add it!